### ISSUE TITLE
🐛 Cannot Clone "layouts/main" after it has executed

### ISSUE THREAD

**Issue by suntong** [gofiber/fiber#814]
**Fiber version**

`v2.0.1`

**Issue description**

**Code snippet**

check out https://github.com/suntong/fiber_demo/blob/master/template-html/main0.go

```
$ go run main0.go 

 ┌───────────────────────────────────────────────────┐
 │                    Fiber v2.0.1                   │
 │               http://127.0.0.1:3000               │
 │                                                   │
 │ Handlers ............. 4  Threads ............ 12 │
 │ Prefork ....... Disabled  PID ............ 304986 │
 └───────────────────────────────────────────────────┘
```

```
$ curl http://localhost:3000/layout
html/template: cannot Clone "layouts/main" after it has executed
```

- It was working before the v2 upgrade.
- fiber/v2 `v2.0.0` seems to be OK too: https://github.com/gofiber/template/runs/1111922299#step:4:11

**Comment by suntong** [comment]
BTW, 

The new fiber console box frame doesn't look good when quoted as code on GH, and it causes freezes in my xterm as well. I really hope it can be changed. 

The v1 logo looks cool, I like it better.

**Comment by iredmail** [comment]
i have same issue with the django template, with “pkger” to embed template files in binary file. It works well before upgrading pongo2 to v4.

Upgrading pango2 to v4 seems breaking my code:
https://github.com/gofiber/template/commit/38576535466db67c3aed53c77eb26b8c116fcb5a

I used `engine := django.New("./templates", ".html")`, and package all files with `pkger -include /static -include /templates`, works fine before upgraded to pongo2 v4. 

but it failed with pongo2 v4. Always get error like this:

```
views: [Error (where: fromfile) in layout.html] unable to resolve template
```

can not find the layout template file defined in template `{% extends "xxx.html" %}`. Tried different file paths, all failed:

- /templates/layout.html
- templates/layout.html
- /layout.html
- layout.html

also tried `engine := django.NewFileSystem(pkger.Dir("/templates"), ".html")`

The (django) template files:

`templates/index.html`:

```
{% extends "layout.html" %}

{% block title %}Index{% endblock %}

{% block body %}
    <p>This is body.</p>
{% endblock %}
```

`templates/layout.html`:

```
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <title>{% block title%}{% endblock %}</title>
    </head>

    <body>
        {% block body %}{% endblock %}
    </body>
</html>
```

**Comment by Fenny** [comment]
Maybe we should downgrade to `django v2` again, the tests seem to pass and I have no personal experience with the django library. Thoughts?

**Comment by iredmail** [comment]
I can help test if it’s downgraded to v2.

**Comment by Fenny** [comment]
Maybe you could help me clone https://github.com/gofiber/template/blob/master/django/django_test.go and see if you could reproduce your issue with a test case.

**Comment by Fenny** [comment]
@iredmail, I downgraded django until we we have a better test case to reproduce your problem.
@suntong the layout problem should also be fixed in `v1.6.2`
<https://github.com/gofiber/template/releases/tag/v1.6.2>
